apps/condo/schema/Billing.js

```diff
@@ -160,6 +204,12 @@ const BillingIntegrationLog = new GQLListSchema('BillingIntegrationLog', {
             schemaDoc: 'Message type. Our clients can use different languages. Sometimes we need to change the text message for the client. The settings for the message texts are in the integration. Ex: WRONG_AUTH_CREDENTIALS',
             type: Text,
             isRequired: true,
+            hooks: {
+                validateInput: ({ resolvedData, fieldPath, addFieldValidationError }) => {
+                    const value = resolvedData[fieldPath]
+                    if (!/^[A-Z_]+$/.test(value)) addFieldValidationError(`${WRONG_TEXT_FORMAT}${fieldPath}] allow only [A-Z_] letters`)
```

Comment:
Can we pick out this regex to named Const. Guess this will improve readability, like `const LATIN_ALPHABET = /^[A-Z_]+$/`

Reply:
move to constants